mscore/pagesettings.h

```diff
@@ -36,15 +38,23 @@ class Navigator;
 class PageSettings : public QDialog, private Ui::PageSettingsBase {
       Q_OBJECT
 
+      static const int PREVIEW_WIDTH  = 0; // This is really NON_HEIGHT, but WIDTH makes it a nice pair
+      static const int PREVIEW_HEIGHT = 1;
```

Comment:
Could be.  If it was 3 items I probably would have made it so.  2 is borderline.